openstudiocore/src/model/ScheduleDay.cpp

```diff
@@ -288,6 +288,10 @@ namespace detail {
       return false;
     }
 
+    if (std::isnan(value) || std::isinf(value)){
+      return false;
+    }
+
```

Comment:
No problem on the position. But this is going to return false without any indication of why. Outputing a message like I did seems like a good thing to me.

Reply:
This is better, I should be better about this